packages/jaeger-ui/package.json

```diff
@@ -85,10 +85,10 @@
     "prop-types": "^15.5.10",
     "query-string": "^6.3.0",
     "raven-js": "^3.22.1",
-    "react": "^16.14.0",
-    "react-circular-progressbar": "^2.0.3",
-    "react-dimensions": "^1.3.0",
-    "react-dom": "^16.14.0",
+    "react": "^18.2.0",
```

Comment:
Wow, that looks suspiciously easy, were there really no breaking changes?

Reply:
The breaking changes of lifecycle etc have been postponed to upcoming versions of React 18 and are still warnings afaik, and that is why its an easier upgrade. As far as I understand and can test the project still can use dependencies which are not yet fully React 18 compatible (like react-vis in this case).

Would love to hear how to move this PR forward and what we can do to test it, what edge cases of libs we think might break.

(I've also submitted changes to migrate from old lifecycle events in react-vis here https://github.com/uber/react-vis/pull/1473)